hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -137,6 +137,16 @@ private static void addDeprecatedKeys() {
   // Resource types configs
   ////////////////////////////
 
+  public static final String NODE_STORE_ROOT_DIR_NUM_RETRIES =
+      YARN_PREFIX + "nodestore-rootdir.num-retries";
```

Comment:
Nit: Since this is an RM config (capacity scheduler feature) I think we could use the yarn.resourcemanager prefix.